tests/test_api.py

```diff
@@ -0,0 +1,36 @@
+from http import HTTPStatus
+
+import pytest
+from fastapi.testclient import TestClient
+from pytest_mock import MockerFixture
+
+from gitlab_project_exporter.gitlab_project import GitlabProject
+from gitlab_project_exporter.main import app
+
+from .conftest import REMOTE_MIRROR_STATUS_KO, REMOTE_MIRROR_STATUS_OK
```

Comment:
Importing from `contest` is bad practice. Better use pytest fixtures.

Reply:
fixed in https://github.com/app-sre/gitlab-project-exporter/commit/baf4ac06dd3c452d48f7310d4c34f14727c1f4c5